frontend/src/components/Metrics/MetricsStep/index.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { Crews } from '@src/components/Metrics/MetricsStep/Crews'
 import { useAppSelector } from '@src/hooks'
 import { selectUsers } from '@src/context/board/jiraVerifyResponse/jiraVerifyResponseSlice'
+import { RealDone } from '@src/components/Metrics/MetricsStep/RealDone'
 
 export const MetricsStep = () => {
   const users = useAppSelector(selectUsers)
-
+  const realDoneOptions = ['DONE', 'CANCELLED']
```

Comment:
I am not very sure about why real done options using `Done` and `Cancelled` as default value. I know this is written in the card. But what if the board don't have a cancelled status? What if the board have more other status in the doneColumn? I mean the status in the done column may vary from boards to boards. I think it probably need to use the data queried from configuration.